yt/utilities/lib/volume_container.pxd

```diff
@@ -16,6 +16,18 @@ A volume container
 
 cimport numpy as np
 
+
+#-----------------------------------------------------------------------------
+# Encapsulates a volume container used for volume rendering.
```

Comment:
Also, since this volume container has a left_edge and a right_edge, is this the same thing as ```lower_left``` and ```upper_right```, i.e. it will always be rectangular in shape?

Reply:
It will be in *index* space, yes.  But the issue we're concerned about here is when index space is logically cartesian but it maps to a non-cartesian space.

Reply:
I'm still unsure of what this means. Can you clarify?

My understanding is the VolumeContainer is our Voxel grid. The left and right edge of the grid are determined by ```left_edge``` and ```right_edge```. Anything outside of that, we don't care about.

How then, do we define a sphere given these parameters?

Reply:
That's what I mean, yes.  Keep in mind that we're juggling two different things here -- the representation of the data in memory, and the representation in physical space.

To represent a sphere, we would have the same components, but they would refer to a different physical coordinate system.  So index 0, 0 ,0, would refer to the cell nearest the pole in a sphere, for instance.

Reply:
Right, so am I correct to say naming the point (0,0,0)  as ```left_edge``` is incorrect, when using r, theta, phi?

Reply:
Sort of.  This is correct:

```
r = 0
theta = 0
phi = 0
```

Reply:
Resolved in Slack.